lib/vdsm/storage/managedvolume.py

```diff
@@ -114,7 +114,7 @@ def attach_volume(vol_id, connection_info, sd_id=None):
                         "rbd, iscsi")
```

Comment:
Regarding the commit message - it will help if you actually show the different variants of the udev rule :-)

Reply:
It's in the PR description, I'll add it to the commit message too